### PR TITLE
Fix long image name handling

### LINKED ISSUE #1655: `pypodman images` doesn't truncate a long image name like `podman images` does.
**Is this a BUG REPORT or FEATURE REQUEST?**:
# **kind bug**

**Description**
Only happens when the image name is pretty long.

**Steps to reproduce the issue:**

1. on remote:
```
# podman images
REPOSITORY                                                                 TAG       IMAGE ID       CREATED        SIZE
localhost/minimal-httpd                                                    latest    cc77fde4759b   23 hours ago   148MB
localhost/init-httpd                                                       latest    b540cee155c4   25 hours ago   222MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8-minimal         8.0-73    2d7409972c5e   10 days ago    89.4MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8/net-snmp        5.8.2-1   bd8911a9446f   3 weeks ago    242MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8/support-tools   8.0-1     1e342da039f6   3 weeks ago    178MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8/rsyslog         8.0-1     1e97bbd0fe95   3 weeks ago    220MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8/sadc            8.0-1     f40386559857   3 weeks ago    217MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8/rhel-tools      8.0-2     be90b6f7718b   4 weeks ago    449MB
brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel8/rhel-init       8.0.0-1   5819904b7fbd   4 weeks ago    202MB
```

**Describe the results you received:**
on host:
```
# pypodman -l root --host 192.168.122.213 images                                
root@192.168.122.213's password:
REPOSITORY                               TAG        IMAGE ID     CREATED      SIZE                       
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 5819904b7fbd Sep 12       192.4 Mi                   
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 be90b6f7718b Sep 17       428.1 Mi                   
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 f40386559857 Sep 19       207.0 Mi                   
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 1e97bbd0fe95 Sep 19       209.4 Mi                   
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 1e342da039f6 Sep 20       169.3 Mi                   
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 bd8911a9446f Sep 21       230.5 Mi                   
brew-pulp-docker01.web.prod.ext.phx2.red 8888/rhel8 2d7409972c5e Oct 05       85.3 MiB                   
localhost/init-httpd                     latest     b540cee155c4 yesterday    211.8 Mi                   
localhost/minimal-httpd                  latest     cc77fde4759b yesterday    141.1 Mi
```
**Describe the results you expected:**
same output as the remote `podman images` shows.

**Additional information you deem important (e.g. issue happens only occasionally):**
always

**Additional environment details (AWS, VirtualBox, physical, etc.):**

### PR DESCRIPTION
**Pull request by jwhonce**
* Fixed issue where podman printed '<none>' and pypodman
  skipped the image
* Fixed issue where port was printed in place of tags

Fixes #1655 

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: contrib/python/pypodman/pypodman/lib/report.py [hunk 1]
```diff
@@ -26,18 +41,24 @@ def __init__(self, columns, heading=True, epilog=None, file=sys.stdout):
...
         self.epilog = epilog
-        self._format = None
 
     def row(self, **fields):
         """Print row for report."""
         if self._heading:
             hdrs = {k: v.display for (k, v) in self._columns.items()}
-            print(self._format.format(**hdrs), flush=True, file=self._file)
+            print(
+                self._formatter.format(self._format_string, **hdrs),
```
Comment (cevich) [inline contrib/python/pypodman/pypodman/lib/report.py]: Ugg, if I see the word 'format' one more time, I'm gunna....

File: contrib/python/pypodman/pypodman/lib/actions/images_action.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def __init__(self, args):
 
         self.columns = OrderedDict({
             'name':
-            ReportColumn('name', 'REPOSITORY', 40),
+            ReportColumn('name', 'REPOSITORY', 0),
```
Comment (cevich) [inline contrib/python/pypodman/pypodman/lib/actions/images_action.py]: Is it a python3-ism that you don't need to supply all the values (missing one for `default` here)?

Comment (jwhonce) [reply contrib/python/pypodman/pypodman/lib/actions/images_action.py]: ReportColumn() defaults missing arguments to None

File: contrib/python/pypodman/pypodman/lib/report.py (hunk 1 above)
Comment (jwhonce) [reply contrib/python/pypodman/pypodman/lib/report.py]: I know, but the base class is Formatter with the method format().  Making up other names seemed more confusing.

File: contrib/python/pypodman/pypodman/lib/actions/images_action.py (hunk 2 above)
Comment (TomSweeneyRedHat) [reply contrib/python/pypodman/pypodman/lib/actions/images_action.py]: luv me my python for this, although sometimes I hate it for this too!

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**